src/replica/duplication/replica_duplicator_manager.cpp

```diff
@@ -127,11 +127,13 @@ void replica_duplicator_manager::update_confirmed_decree_if_secondary(decree con
 
```

Comment:
If `request.__isset.confirmed_decree = false`, what's value of `confirmed`? is it equal to 0 or below 0?

Reply:
Oh, I thought the `__isset.confirmed_decree` is `request.confirmed_decree` before. but it actually is `whether set confirmed_decree`:
```c++
void group_check_request::__set_confirmed_decree(const int64_t val) {
  this->confirmed_decree = val;
__isset.confirmed_decree = true;
}
```

however, I find the value is always `true`:  https://github.com/XiaoMi/rdsn/blob/5c8de19867385cfb54d7da22860b85485055d801/src/replica/replica_check.cpp#L104

So I think revert or keep is same